.github/workflows/oclif-release.yml

```diff
@@ -76,7 +80,11 @@ jobs:
     steps:
       - name: Checkout cli repo
         uses: actions/checkout@v2
+      - name: Extract branch name
+        id: extract_branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
       - name: Invoke Scoop workflow
+        if: ${{steps.extract_branch.outputs.branch == 'main'}}
```

Comment:
Does this mean for rc, we won't do a scoop and platform executables release? Currently, we don't.

Reply:
Currently I am merging the release candidate changes in `release-feature-branch` into `main` by adding some conditions so that it should not effect the main release and we are doing release candidate for NPM, Docker and Homebrew in `release-feature-branch.`